src/Attachment.tsx

```diff
@@ -26,17 +60,20 @@ const Youtube = (props: {
 }) =>
     <iframe
         type="text/html"
-        src={ new URI()
-            .domain(YOUTUBE_DOMAIN)
-            .subdomain("")
-            .port("")
-            .segment(["embed", props.embedId])
-            .search({
-                "modestbranding": 1,
-                "loop": props.loop ? 1 : 0,
-                "autoplay": props.autoPlay ? 1 : 0
-            })
-            .toString()
+        src={
+            buildUrl(
+                [
+                    "https://",
+                    YOUTUBE_DOMAIN,
+                    "/embed/",
+                     props.embedId
+                ].join(''),
```

Comment:
I'd rather use string interpolation here and below, as it's much clearer (and, in latest browsers, now as faster or faster than array join)., e.g. \``https://${YOUTUBE_DOMAIN}/embed/${props.embedId}`\`